### ISSUE TITLE
feature-request/ show shimmer while scroll fast

### ISSUE THREAD

**Issue by ydv0121** [marcocesarato/react-native-big-list#7]

**Comment by marcocesarato** [comment]
Hi @ydv0121 , this isn't possibile because the blank space is a space from the last item to the end of the scrollview (not visibile) and if we sobstitute it with items the performance drops but there is a possible **workaround** to make an *illusion* of a shimmer. This workaround is to add a Image as repeated background. So in the end users should choice the image to repeat (like an image with a single stripe) or a component similar (like a react-native-svg that do the same thing)

Here and example with a repeated image:

https://user-images.githubusercontent.com/36447518/122114731-e967c000-ce23-11eb-87e4-f399f86e6250.mp4

If this is a good solution for you I can implement it.

**Comment by ydv0121** [comment]
@marcocesarato yes it's great instead of white spaces in big list.
Thanks

**Comment by marcocesarato** [comment]
Added on `1.3.8` as experimental feature. I have encountered problems with screen resolutions and image resolution so this feature needs to be refined.

Using `placeholder={true}` you will see the default placeholder.
Using `placeholderImage={require('./assets/yourImage.png')}` you can set your pattern of image to repeat.
Using `placeholderComponent` you can set a component like an svg.

Let me know if you run into any issues and how this feature can be improved.

**Comment by marcocesarato** [comment]
I saw some bugs on android, so it need some improvements.

**Comment by ydv0121** [comment]
yes there is also UI glitch. so it's not good idea to show shimmer in white space?

**Comment by marcocesarato** [comment]
> yes there is also UI glitch. so it's not good idea to show shimmer in white space?

Probably the best solution is to use and SVG pattern or find a similar solution, but at the moment it's better don't implement an SVG pattern as default (because need a to add a new dependency) until found a new solution.

**Comment by dougg0k** [comment]
You could give the option and recommend the svg library to whoever is using and want shimmer. Maybe even restrict and make it required so no glitchs appear.

**Comment by ydv0121** [comment]
i am trying to integrate this module with biglist https://github.com/tomzaku/react-native-shimmer-placeholder
